tests/config.c

```diff
@@ -128,13 +130,12 @@ platform_status
 config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
 {
    uint64 i;
-   uint8  cfg_idx;
    for (i = 0; i < argc; i++) {
       // Don't be mislead; this is not dead-code. See the config macro expansion
       if (0) {
          config_set_uint64("page-size", cfg, page_size)
          {
-            for (cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+            for (uint8 cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
```

Comment:
Let's be consistent with the types here, even though it doesn't really matter. I generally default to `uint64` in this case, so I would prefer that here.

Reply:
Well ... I was just applying the type that was used for this variable's definition on deleted old line L131.
(No change; left it as uint8.)